vars/list_preseeds.yml

```diff
@@ -0,0 +1,10 @@
+---
+list_preseeds:
+    - name: "Debian Stable"
+      description: "KG for: ERP 17.12, Stretch"
```

Comment:
This is not specific to ERP 17.12. I think we need to name it more wisely, but I'm not wise enough to come up with a good name. So for now, I'd just call it "Debian" and "CentOS" and we can add other names when we have more than two in the future.